dotcom-rendering/src/web/components/ImageComponent.tsx

```diff
@@ -379,7 +381,15 @@ export const ImageComponent = ({
 				{isMainMedia && (
 					// Below tablet, main media images show an info toggle at the bottom right of
 					// the image which, when clicked, toggles the caption as an overlay
-					<Hide when="above" breakpoint="tablet">
+					<Hide
+						when="above"
+						breakpoint={
+							format.design === ArticleDesign.LiveBlog &&
+							isMainMediaTest
+								? 'desktop'
+								: 'tablet'
+						}
+					>
```

Comment:
This seems to be adding a lot of complexity, is this really needed? It would be great is both the code and the designs were simpler as complexity on both sides has costs which tend to endure for years and these costs should be weighed against the benefits to adding this variation